cli/cli_app.go

```diff
@@ -617,9 +618,10 @@ func shouldRunHook(hook config.Hook, terragruntOptions *options.TerragruntOption
 	//resolves: https://github.com/gruntwork-io/terragrunt/issues/459
 	//by helping to filter out nil errors that were acting as false positives
 	//for the len(previousExecErrors) == 0 check that used to be here
```

Comment:
NIT: This comment is no longer relevant, so I think we can remove it.

Reply:
removed irrelevant part of the comment